homeassistant/components/tolosauna/config_flow.py

```diff
@@ -0,0 +1,96 @@
+"""Config flow for tolosauna."""
+
+import logging
+from typing import Any, Dict, Optional
+
+from tololib import ToloClient
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow
+from homeassistant.const import CONF_HOST
+from homeassistant.data_entry_flow import FlowResult
+
+from ...helpers.device_registry import format_mac
+from ..dhcp import IP_ADDRESS, MAC_ADDRESS
+from .const import DEFAULT_NAME, DEFAULT_RETRY_COUNT, DEFAULT_RETRY_TIMEOUT, DOMAIN
+
+DATA_SCHEMA_USER = vol.Schema({vol.Required(CONF_HOST): str})
+
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class ToloSaunaConfigFlow(ConfigFlow, domain=DOMAIN):
+    """ConfigFlow for TOLO Sauna."""
+
+    VERSION = 1
+
+    def __init__(self) -> None:
+        """Initialize ToloSaunaConfigFlow."""
+        self._discovered_host: Optional[str] = None
+
+    @staticmethod
+    def _check_device_availability(host: str) -> bool:
+        client = ToloClient(host)
+        result = client.get_status_info(
+            resend_timeout=DEFAULT_RETRY_TIMEOUT, retries=DEFAULT_RETRY_COUNT
+        )
+        if result is None:
+            return False
+        else:
+            return True
```

Comment:
```suggestion
        if result is None:
            return False
        return True
```

Reply:
Replaced with `return result is not None`, which is more Python style.